### ISSUE TITLE
Change pointer when hovering over a link

### ISSUE THREAD

**Issue by PankajBhojwani** [microsoft/terminal#7563]
# Description of the new feature/enhancement
When the user hovers over a link and presses ctrl, change the pointer to the 👆 icon

**Comment by lubokkanev** [comment]
Also please support ctrl-clicking on the link to open it in the browser.

**Comment by zadjii-msft** [comment]
@lubokkanev it already does that - see #7251, #7691

**Comment by lubokkanev** [comment]
Great! #7691 is what I meant but it doesn't seem to be working for me.
I'm on Windows Terminal Preview Version: 1.4.2652.0. Should it be on?

**Comment by zadjii-msft** [comment]
Nope, that PR is still open - in 1.4 you'll need an application that knows how to manually tell the Terminal about a hyperlink. Hopefully, #7691 should land in v1.5.
